source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -90,7 +90,7 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::jwt>,
   Http::HeaderMap* headers_{};
   // The active span for the request
   Tracing::Span* parent_span_{&Tracing::NullSpan::instance()};
-  // the callback function to set payload
+  // The callback function called to set the extracted payload and header from a verified JWT.
```

Comment:
`set_payload_cb_` should have a more generic name as it now sets both payload and header.

Reply:
Updated.